src/index.js

```diff
@@ -75,7 +75,7 @@ var MaskedInput = React.createClass({
         this._updateInputSelection()
       }
     }
-    if (this.props.onChange) {
+    if (typeof this.props.onChange !== 'undefined') {
```

Comment:
`if (this.props.onChange)` is fine here. They provide the same level of safety here (and the rest of the cases below).